server/db/foods.js

```diff
@@ -3,6 +3,36 @@ const connection = require('./connection')
 function getFoods (db = connection) {
   return db('foods')
 }
+
+function editFood (food, db = connection) {
```

Comment:
Can this be moved below the get foods/food functions? So our simple easy to read is first. Thanks